cells/patroni/nomadCharts/default.nix

```diff
@@ -58,8 +60,9 @@ in
             operator = "is_set";
           }
           {
-            operator = "distinct_hosts";
-            value = "true";
+            operator = "distinct_property";
```

Comment:
I think `distinct_hosts` will fingerprint the nomad client and ensure avoidance of deployment overlap, whereas attr.unique.hostname is not a fingerprint and may be collisional.  Example: prem machines where VMs are supplied with a clone from a standard OS image and a non-unique hostname.  Is there an advantage to switching?

Reply:
We are running multiple namespaces containing multiple patroni instances on the same machines. `distinct_hosts` disregards the fact that these patroni instances are running in separate namespaces and prevents them from being scheduled https://github.com/hashicorp/nomad/issues/9792

Reply:
Ah, I see, thanks!